man/fi_peer.3.md

```diff
@@ -286,6 +286,91 @@ on the local node.
    creates a reference between the peer_cq object and its own cq.
 ```
 
+# PEER COUNTER
+
+The peer counter defines a mechanism by which a peer provider may increment value or error into
```

Comment:
Please check line lengths throughout this new section.  There are several areas that are too long.

Reply:
New revision limits each line <= 80 characters.